ansible/playbooks/service/sssd.yml

```diff
@@ -0,0 +1,37 @@
+---
+# Copyright (C) 2019 Maciej Delmanowski <drybjed@gmail.com>
+# Copyright (C) 2019 DebOps <https://debops.org/>
+# Copyright (C) 2021 David Härdeman <david@hardeman.nu>
```

Comment:
Please update the copyright information in the new files, drop my copyright (I haven't even touched these files yet :-) ) and change the year of the DebOps copyright to 2021. You can also move the DebOps copyright under your own, since you are the initial contributor.

Reply:
I think the copyright notices are largely correct. Almost every single file in the new service is based on copy-pasta from other existing roles (just as an example, `roles/sssd/templates/etc/sssd/sssd.conf.j2` is based on `roles/rsyslog/templates/etc/rsyslog.conf.j2` with a diff that's about four lines, ignoring comments).

In those cases, I've kept the original copyright notices and added my own since it's a derivative work. Also not clear to me that the order of the copyright notices is relevant, but I'd be happy to change them if it's important.

Reply:
In that case move the DebOps copyright under personal copyrights and update the year to 2021 since the code will be included in the project.

Reply:
Ok, I think I've updated all the copyright declarations.